neofetch

```diff
@@ -30,7 +30,13 @@
 
 version=7.0.0
 
-bash_version=${BASH_VERSION/.*}
+if [[ "${BASH_VERSINFO[0]}" ]]; then
+    bash_version=${BASH_VERSINFO[0]}
+else
+    bash_version=5
+    shopt -s eval_unsafe_arith
+fi
```

Comment:
It's probably easier to just do `bash_version=${BASH_VERSINFO[0]:-5}` and then run the shopt with `&>/dev/null` (always).

Reply:
Question, will it be difficult for us to remove the shopt entirely? What needs to be done?

Reply:
I actually haven't had any issues without it, but I noticed that you do an `unset -v "gpus[0]"` on a certain condition in the script, which is invalid in OSH without `eval_unsafe_arith`.